pkg/identityserver/store/gateway.go

```diff
@@ -63,12 +64,16 @@ type Gateway struct {
 	UpdateLocationFromStatus bool `gorm:"default:false not null"`
 
 	Antennas []GatewayAntenna
+
+	LBSLNSSecret []byte `gorm:"type:BYTEA;column:lbs_lns_secret"`
```

Comment:
Consider a default value here - otherwise old databases will complain that they cannot migrate already existing gateways. 

_I forgot this at least twice already_

Reply:
Isn't that only for `not null` fields @htdvisser ?

Reply:
Yeah. Nullable columns have null as default "value".